utils/ExampleRunner/template-config.js

```diff
@@ -45,6 +45,11 @@ module.exports = {
             '../../../node_modules/@cornerstonejs/dicom-image-loader/dist/dynamic-import',
           to: '${destPath.replace(/\\/g, '/')}',
         },
+        {
+          from:
+          '../../../node_modules/dicom-microscopy-viewer/dist/dynamic-import/index.worker.min.worker.js',
```

Comment:
We should consider randomizing the UID in dicomImageLoader. This is important because conflicts may arise if another library also has index.worker.js, since we copy this to serve.

Reply:
I'd really prefer to change this to just load from a nested path, but I can't figure out the webpack mumbo jumbo to enable that.  Even better would be to direct load as a script external.  I THINK I know how to do that one.

Reply:
Loads on nested path.  Does not need randomization any longer.